simulation/vis_simulate.pro

```diff
@@ -40,7 +40,10 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
     beam_arr=beam_image_cube(obs,psf, n_freq_bin = n_freq,/square)
     for freq_i=0,n_freq-1 do begin
       beam2_xx_image[*,*, freq_i] = Temporary(*beam_arr[0,freq_i])
-      beam2_yy_image[*,*, freq_i] = Temporary(*beam_arr[1,freq_i])
+    ;  beam2_yy_image[*,*, freq_i] = Temporary(*beam_arr[1,freq_i])
+      if n_pol gt 1 THEN beam2_yy_image[*,*, freq_i] = Temporary(*beam_arr[1,freq_i]) $
+      else beam2_yy_image[*,*,freq_i] = beam2_xx_image[*,*,freq_i]
+
```

Comment:
(GitHub strips the spaces I added for indentation, so please do use indentation inside if statements!)